doc/minversions.rst

```diff
@@ -8,101 +8,123 @@ the rest of the Scientific Python ecosystem, and therefore:
 * Support for core package dependencies (NumPy/Pandas/Xarray) be dropped 2 years after their initial release.
 
 .. list-table::
-    :widths: 25 30 15 20 15
+    :widths: 25 15 15 15 15 15
     :header-rows: 1
+    :align: center
 
     * - PyGMT Version
-      - Documentation
       - GMT
       - Python
       - NumPy
-    * - `Dev <https://github.com/GenericMappingTools/pygmt/milestones>`_ (upcoming release)
-      - `Dev Documentation <https://www.pygmt.org/dev>`_ (reflects `main branch <https://github.com/GenericMappingTools/pygmt>`_)
+      - Pandas
+      - Xarray
+    * - `Dev <https://github.com/GenericMappingTools/pygmt/milestones>`_ * [`Docs <https://www.pygmt.org/dev>`__]
```

Comment:
Is there a need that the first link ends with one underscore (`_`) but the second link with two underscores (`__`)?
```suggestion
    * - `Dev <https://github.com/GenericMappingTools/pygmt/milestones>`_ * [`Docs <https://www.pygmt.org/dev>`_]
```

Reply:
See https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use for explanations.

For "Docs", we have to use `__` to make the links private.

Reply:
Ah, interesting. I did not know this. Thanks for sharing this link!